src/main/resources/liquibase/crop_changelog/24_5_0.xml

```diff
@@ -104,4 +104,598 @@
+            </sqlCheck>
+        </preConditions>
+        <comment>Add scale Packing status categories</comment>
+        <sql dbms="mysql" splitStatements="true">
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1030, 'Packing status categories', 'Packing status categories', null, 0, 0);
+        </sql>
+    </changeSet>   
+		
+	<changeSet id="v24.5.0-5" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Packing_Status" and cv_id = 1040 ;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add variable for Packing_Status</comment>
+        <sql dbms="mysql" splitStatements="true">
+            set @methodID = (select cvterm_id from cvterm where name = 'Described' AND cv_id = 1020);
+            set @scaleId = (select cvterm_id from cvterm where name = 'Packing status categories' AND cv_id = 1030);
+			set @propertyId = (select cvterm_id from cvterm where name = 'Packing status' AND cv_id = 1010);
+
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1040, 'Packing_Status', 'Packing status of the plot in the seed prep workflow.', null, 0, 0);
+           
+            set @variableId = (select cvterm_id from cvterm where name = 'Packing_Status' AND cv_id = 1040);
+			set @propertyId = (select cvterm_id from cvterm where name = 'Packing status' AND cv_id = 1010);
+		
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1200, @variableId, @propertyId);
+
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1210, @variableId, @methodId);
+
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1220, @variableId, @scaleId);
+			
+            INSERT INTO cvtermprop(cvterm_id, type_id, value, rank)
+            VALUES (@variableId, 1800, 'Trait', 0);
+			
+			INSERT INTO cvtermprop (cvterm_id, type_id, value, rank) 
+			VALUES (@variableId, 8048, (SELECT DATE_FORMAT(CURRENT_TIMESTAMP(), '%Y-%m-%dT%T.000Z')), 0);
+	
+        </sql>
+    </changeSet>	
+	
+	<changeSet id="v24.5.0-6" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                select count(*) from cv where name = "Packing_Status" ;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add categories for variable Packing_Status</comment>
+        <sql dbms="mysql" splitStatements="true">
+			set @variableId = (select cvterm_id from cvterm where name = "Packing_Status" AND cv_id = 1040);
+			set @scaleId = (select cvterm_id from cvterm where name = "Packing status categories" AND cv_id = 1030);
+			
+			INSERT INTO cv ( name, definition)
+			VALUES (@variableId, "Packing_Status");
```

Comment:
@rvtulod The scaleID should be stored in cv.name, not the variableId. Should be corrected to 
**INSERT INTO cv ( name, definition)
VALUES (@scaleId, "Packing status categories");**